Purchases/SwiftSources/LocalReceiptParsing/BasicTypes/InAppPurchase.swift

```diff
@@ -0,0 +1,68 @@
+//
+// Created by Andrés Boedo on 7/29/20.
+// Copyright (c) 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+enum InAppPurchaseAttributeType: UInt {
+    case quantity = 1701,
+         productId = 1702,
+         transactionId = 1703,
+         purchaseDate = 1704,
+         originalTransactionId = 1705,
+         originalPurchaseDate = 1706,
+         productType = 1707,
+         expiresDate = 1708,
+         webOrderLineItemId = 1711,
+         cancellationDate = 1712,
+         isInTrialPeriod = 1713,
+         isInIntroOfferPeriod = 1719,
+         promotionalOfferIdentifier = 1721
```

Comment:
`promotionalOfferIdentifier`, `isInTrialPeriod` and `productType` seem to work and are useful, but they aren't documented. 
I made them optional since they're not documented, just in case they change in the future.